### PR TITLE
implemented first version of adding event to tum calendar

### PR DESCRIPTION
**Pull request by saileru**
Events from the ticket sale can now be added to the TUM Calendar.

Open question: When the users click on the date field they should probably be asked if they want to add the event to the TUM calendar or the external calendar app. What are your opinions on that?

I also discovered a weird bug. Event titles which include multiple upper case letters are not or not completely displayed in the calendar. For example, "TUNIX 2018" is not displayed at all.
e.g."abc", "Abc", "Ab Ab", "A-Ab" work correctly. Problems occur i.a. in the following examples:
"A" ->""
"A-A" ->"A"

Any thoughts/experience on that? Should I create a bug ticket for that?
I could also reproduce this error on the master branch.

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java [hunk 1]
```diff
@@ -175,14 +183,28 @@ private void buyTicket() {
...
-                .putExtra(CalendarContract.Events.TITLE, event.getTitle())
-                .putExtra(CalendarContract.Events.DESCRIPTION, event.getDescription())
-                .putExtra(CalendarContract.Events.EVENT_LOCATION, event.getLocality())
-                .putExtra(CalendarContract.Events.AVAILABILITY, CalendarContract.Events.AVAILABILITY_FREE);//Indicates that this event is free time and will not conflict with other events.
+        // TODO: choose between TUM Calendar and external calendar app
+        boolean tumCalendar = true;
+        Intent intent;
+        if(tumCalendar){
+            intent = new Intent(context, CreateEventActivity.class);
+            intent.putExtra(Const.EVENT_EDIT, false);
```
Comment (schulzalexander) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java]: We should also consider the case where the user has already added the event to the calendar, then the edit property should be true right? Or alternatively, we could just take the user to the corresponding day in the calendar.

**Comment by schulzalexander** [comment]
@saileru since the naming bug seems to be a problem of the calendar itself, I'd suggest creating a bug for that.

**Comment by saileru** [comment]
Issue for naming bug: #970

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java (hunk 1 above)
Comment (saileru) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java]: I am not sure if this is feasible. How can we reliably check if the event has already been created? The user might have manually adjusted time and/or title.

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java [hunk 2]
```diff
@@ -175,14 +183,28 @@ private void buyTicket() {
...
+            intent.putExtra(Const.EVENT_TITLE, event.getTitle());
+            intent.putExtra(Const.EVENT_COMMENT, event.getDescription());
+            intent.putExtra(Const.EVENT_START, DateUtils.getDateTimeString(event.getDate()));
+            intent.putExtra(Const.EVENT_END, DateUtils.getDateTimeString(
+                    new Date(event.getDate().getTime() + eventDuration)));
+        } else{
```
[the remark this answers is not in the document]
Comment (saileru) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java]: Will do!

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java [hunk 3]
```diff
@@ -45,6 +48,8 @@
 
     private final CompositeDisposable disposable = new CompositeDisposable();
 
+    private long eventDuration = 7200000;
```
[the remark this answers is not in the document]
Comment (saileru) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java]: Correct, I wil add a TODO in the next commit.

**Comment by saileru** [comment]
Here are screenshots of the "Export Event" Dialog for German and English

screenshot_20180707-175358
screenshot_1530978978

**Comment by saileru** [comment]
New layout: buttons below each other
screenshot_20180708-193136

**Pull request merged**